api/src/pcapi/core/finance/api.py

```diff
@@ -2052,14 +2054,18 @@ def get_collective_data(query: BaseQuery) -> BaseQuery:
                 models.Invoice.reimbursementPointId,
                 models.PricingLine.category,
                 educational_models.EducationalDeposit.ministry,
+                educational_models.EducationalInstitutionProgram.id,
             )
             .with_entities(
                 models.Invoice.id,
                 models.Invoice.date.label("invoice_date"),
                 models.Invoice.reference.label("invoice_reference"),
                 models.Invoice.reimbursementPointId.label("reimbursement_point_id"),
                 models.PricingLine.category.label("pricing_line_category"),
-                educational_models.EducationalDeposit.ministry.label("ministry"),
+                sqla.func.coalesce(
+                    educational_models.EducationalInstitutionProgram.label,
+                    educational_models.EducationalDeposit.ministry.cast(sqla.String),
+                ).label("ministry"),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏ C'est là qu'il y aura la mention de `MEG` ?